web-console/knikubevirt/link-between-PVC-VMdisk/link-between-PVC-VMdisk.md

```diff
@@ -4,12 +4,12 @@ In some cases, a disk is being backed by a Kubernetes Persistent Volume Claim (P
 In the OpenShift Console UI, we list these PVCs out for the user to view and edit.
 In our Virtualization UI, we should consider how we might show that a Disk is backed by a PVC and then link over to that PVC.
```

Comment:
Since we have a path forward on eliminating our fork, I think we can get rid of the reference to our Virtualization UI. This will simply be focused on linking to the PVC details within the OpenShift Console.